webapp/channels/src/components/backstage/components/backstage_list.tsx

```diff
@@ -85,6 +116,19 @@ const BackstageList = ({searchPlaceholder = localizeMessage('backstage_list.sear
         );
     }
 
+    const hasFilter = filter.length > 0;
+    const {startCount, endCount, total, isFirstPage, isLastPage} = getPaging(remainingProps, childCount, hasFilter);
+    let childrenToDisplay = childCount > 0 ? children.slice(startCount - 1, endCount) : children;
```

Comment:
Interesting idea. Based on your other suggestion, I initialized children as an empty array to ensure that "length" is always defined. So there is no need to explicitly check again that children is an array (since it is initialized as an empty array), right?